sleap/nn/evals.py

```diff
@@ -135,7 +135,8 @@ def compute_oks(
     points_gt: np.ndarray,
     points_pr: np.ndarray,
     scale: Optional[float] = None,
-    stddev: float = 0.025,
+    use_cocoeval: Optional[bool] = True,
```

Comment:
```suggestion
    use_cocoeval: bool = True,
```
[`Optional`](https://docs.python.org/3/library/typing.html#typing.Optional) indicates that `use_cocoeval` can also be `None`.